text/0000-00-00-coprocessor-write.md

```diff
@@ -0,0 +1,154 @@
+# Coprocessor Write
+
+## Summary
+
+This RFC intends to extend the ability of coprocessor to support writing. The user can send a _single_ coprocessor request to do a TiDB query (a.k.a. a DAG request) first and then perform a write request (like prewrite or pessimistic lock) with the result of the previous TiDB query.
+
+## Motivation
+
+Currently, all write requests are initiated from a TiKV client. To update some data, the user typically reads from TiKV first and prepares the data to write. And then, it sends a write request to TiKV. The whole process includes at least two RPCs between the client and TiKV. There is usually extra cost of encoding and decoding.
+
+However, in multiple cases, we can move some of the logic from the client to TiKV to reduce one RPC and save encoding/decoding cost.
+
+### Simple updates
```

Comment:
I think we could make it more detailed about what scenarios we are about to solve with this RFC, how should we extend it if we want to solve more.